cleanlab/filter.py

```diff
@@ -398,6 +411,80 @@ class 0, 1, ..., K-1.
     return label_issues_mask
 
 
+def _find_label_issues_multilabel(
+    labels,
+    pred_probs,
+    return_indices_ranked_by=None,
+    rank_by_kwargs={},
+    filter_by="prune_by_noise_rate",
+    frac_noise=1.0,
+    num_to_remove_per_class=None,
+    min_examples_per_class=1,
+    confident_joint=None,
+    n_jobs=None,
+    verbose=False,
+):
```

Comment:
note we are using type annotations in the package going forward, so all function signatures  should declare type of each arg as well as the return type. See for example:  https://github.com/cleanlab/cleanlab/blob/master/cleanlab/token_classification/filter.py

(unfortunately we haven't updated every file to comply with the type annotations yet, but all files in token_classification are compliant and serve as good references)